Sources/Stevenson/JiraService.swift

```diff
@@ -60,7 +60,7 @@ public extension JiraService {
 
 // MARK: Jira Common Field Types
 
-public extension JiraService {
+extension JiraService {
```

Comment:
[ ❔] Have we migrated the project to Xcode 10.2 yet? I think Xcode 10.2 introduce warnings for useless duplicate ACLs in declarations now, so would have helped spotting those :)

Reply:
yes, that's why I went with fixing them now